notebooks/245-llm-chatbot/245-llm-chatbot.ipynb

```diff
@@ -0,0 +1,966 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #46.                {"input_ids": torch.ones((1,2), dtype=torch.long), "past_key_values": outs.past_key_values, "attention_mask": torch.ones((1,12), dtype=torch.long)}, 
</pre><p>This is a clever way to get <code>past_key_values </code>example for the conversion, but for the user it may be not obvious </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1207/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>